commit cbb64c08fea1a81a3f8ab5fccf4d88544fba4a92 (refs/changes/24/2524/1)
Author: Julien Gilli <julien.gilli@joyent.com>
Date:   2017-09-08T10:51:52-04:00 (2 years, 1 month ago)
    
    DOCKER-863: Create shared volume when mounting nonexistent volume
    VOLAPI-55 After enabling volapi, a sdcadm update without volapi fails
    WORKFLOW-219 fix configuration file template
    TOOLS-1824 sdcadm update-other should not do anything for VOLAPI
    TOOLS-1830 replace sdcadm experimental nfs-volumes with cloudapi-nfs-volumes/docker-nfs-volumes subcommands

diff --git a/package.json b/package.json
index 18faf99..970412d 100644
--- a/package.json
+++ b/package.json
@@ -17,7 +17,7 @@
     "uuid": "3.0.1",
     "async": "0.1.22",
     "vasync": "1.6.4",
-    "sdc-clients": "10.2.0",
+    "sdc-clients": "git+ssh://git@github.com:joyent/node-sdc-clients.git#tritonnfs",
     "verror": "1.6.0",
     "imgmanifest": "git+https://github.com/joyent/node-imgmanifest.git#e21ea53",
     "lstream": "0.0.4"
diff --git a/sapi_manifests/workflow/template b/sapi_manifests/workflow/template
index aa14f32..4a6d36c 100644
--- a/sapi_manifests/workflow/template
+++ b/sapi_manifests/workflow/template
@@ -82,6 +82,7 @@
                 "lstream": "lstream"
             },
             "ufdsUrl": "ldaps://{{{UFDS_SERVICE}}}",
+            "volapiUrl": "http://volapi.{{{datacenter_name}}}.{{{dns_domain}}}",
             "ufdsDn": "{{{ufds_ldap_root_dn}}}",
             "ufdsPassword": "{{{ufds_ldap_root_pw}}}",
             "dapiUrl": "http://{{{DAPI_SERVICE}}}",
